tests/element_test_vectors.rs

```diff
@@ -649,20 +652,12 @@ mod native_element_tests {
                 // Requires importing shared symbol tables
                 "ion-tests/iontestdata/good/item1.10n",
                 "ion-tests/iontestdata/good/localSymbolTableImportZeroMaxId.ion",
-                "ion-tests/iontestdata/good/notVersionMarkers.ion",
                 // Requires importing shared symbol tables
                 "ion-tests/iontestdata/good/testfile35.ion",
-                // This test has a Decimal with an enormous exponent. We'd need to change
-                // DecodedInt to store an Integer instead of an i64.
-                "ion-tests/iontestdata/good/typecodes/T5.10n",
                 // These files are encoded in utf16 and utf32; the reader currently assumes utf8.
                 "ion-tests/iontestdata/good/utf16.ion",
                 "ion-tests/iontestdata/good/utf32.ion",
                 // NON-EQUIVS
-                // These tests fail because the ion-c writer bindings don't support writing -0.0
-                "ion-tests/iontestdata/good/non-equivs/decimals.ion",
-                "ion-tests/iontestdata/good/non-equivs/floatsVsDecimals.ion",
-                // -----
```

Comment:
🗺️ Now that we're not using the `ion-c` writer for the native tests, the failures caused by the `ion-c` writer are no longer a problem.

Reply:
🎊